README.md

```diff
@@ -12,37 +12,35 @@ Purpose
 
 Bio-Formats' primary purpose is to convert proprietary microscopy data into 
 an open standard called the OME data model, particularly into the OME-TIFF 
-file format. See the [statement of purpose](http://www.openmicroscopy.org/site/support/bio-formats/about/index.html) 
+file format. See the [statement of purpose](docs/sphinx/about/index.rst)
 for a thorough explanation and rationale.
 
 
 Supported formats
 -----------------
 
 Bio-Formats supports [more than a hundred file
-formats](http://www.openmicroscopy.org/site/support/bio-formats/supported-formats.html).
+formats](docs/sphinx/supported-formats.rst).
```

Comment:
The problem with replacing this with a relative link is that GH really doesn't render the table nicely and you can't sort it.
And actually, even with the pages GH does render, the internal links on those pages don't work so over all I think using relative links in this README is not ideal.